src/display/editor/annotation_editor_layer.js

```diff
@@ -84,6 +84,10 @@ class AnnotationEditorLayer {
     this.#uiManager.addLayer(this);
   }
 
+  isEmpty() {
```

Comment:
How about the following, since I'm kind of partial to getters for these types of methods :-)
```suggestion
  get isEmpty() {
```